client/src/main/java/io/split/client/SplitClientImpl.java

```diff
@@ -106,42 +109,77 @@ public String getTreatment(Key key, String split, Map<String, Object> attributes
         return getTreatment(key.matchingKey(), key.bucketingKey(), split, attributes);
     }
 
+    @Override
+    public SplitResult getTreatmentWithConfig(String key, String split) {
+        return getTreatmentWithConfigInternal(key, null, split, Collections.<String, Object>emptyMap());
+    }
+
+    @Override
+    public SplitResult getTreatmentWithConfig(String key, String split, Map<String, Object> attributes) {
+        return getTreatmentWithConfigInternal(key, null, split, attributes);
+    }
+
+    @Override
+    public SplitResult getTreatmentWithConfig(Key key, String split, Map<String, Object> attributes) {
+        if (key == null) {
+            _log.error("getTreatment: you passed a null key, the key must be a non-empty string");
```

Comment:
Is the word `getTreatment` prompted also when you call `getTreatmentWithConfig`? We decided to display `getTreatmentWithConfig` in the case that user call that method